pkg/kubelet/dockershim/helpers_linux.go

```diff
@@ -62,7 +62,8 @@ func getSeccompDockerOpts(seccompProfile string) ([]dockerOpt, error) {
 		return nil, fmt.Errorf("unknown seccomp profile option: %s", seccompProfile)
 	}
 
-	fname := strings.TrimPrefix(seccompProfile, "localhost/") // by pod annotation validation, name is a valid subpath
+	// get the full path of seccomp profile when prefixed with 'localhost/'.
+	fname := strings.TrimPrefix(seccompProfile, "localhost/")
```

Comment:
Not sure you end up with an absolute path here. `TrimPrefix` will cut the `/` too, and you're left with a path that has no leading `/`. From what I can tell, `filepath.FromSlash` does not add a leading `/`. https://play.golang.org/p/_A6DBoSG1V

Reply:
@mtaufen yep, it doesn't add a leading '/'. And there are indeed issues about seccomp path.

`localhost/` is missing because of following code:
https://github.com/kubernetes/kubernetes/blob/72a2b0be6959191587470616148afdf2235b04ef/pkg/kubelet/kuberuntime/helpers.go#L278-L282

will add a new commit to fix this.